crawl-ref/source/mon-death.cc

```diff
@@ -1039,6 +1040,22 @@ static void _infestation_create_scarab(monster* mons)
     infestation_death_fineff::schedule(mons->pos(), mons->name(DESC_THE));
 }
 
+static void _pharaoh_ant_bind_souls(monster *mons)
```

Comment:
This is an interesting effect. It says "kill me last". For that to be an interesting tension, rather than "oh, kill this unthreatening monster last", pharaoh ants need to have a strong attack and poor defenses. (Thus, normally something you'd *want* to kill first.)

I think they might have a bit too much HP and a bit too little damage for that at present. What do you think?